core/src/test/java/io/lionweb/utils/PartitionChunkValidatorTest.java

```diff
@@ -100,7 +100,7 @@ public void testMultipleMissingNodes() {
     ValidationResult result = validator.validate(chunk);
 
     assertFalse(result.isSuccessful());
-    assertEquals(4, result.getIssues().size()); // All 4 missing nodes should be reported
+    assertEquals(5, result.getIssues().size()); // All 4 missing nodes should be reported
```

Comment:
(comment's not updated)

Reply:
Removing it: one less thing to maintain :D